src/Cli/AzureCliLogin.ts

```diff
@@ -34,6 +34,13 @@ export class AzureCliLogin {
         await this.executeAzCliCommand(["--version"], true, execOptions);
         core.debug(`Azure CLI version used:\n${output}`);
 
+        try {
+            await this.executeAzCliCommand(["logout"], true, execOptions);

```

Comment:
Perhaps `az account clear` for a cleaner environment?

Reply:
I suppose `az logout` is enough for signing out the current active subscription. And the following steps will use the subscription set in azure/login. There may be a situation that customer needs to switch the active subscription by manually running `az account set`.

Reply:
In automation, `az account clear` may be a better choice for a cleaner environment. I will modify the code.